src/pages/LandingPage.js

```diff
@@ -51,11 +52,11 @@ const makeRightArrowWithBackgroundIcon = () =>
     [icon('arrowRight', { color: 'white' })]
   );
 
-const makeCard = (link, title, body) =>
+const makeCard = _.map(({ link, title, body, linkPathParams, linkQueryParams }) =>
   h(
     Clickable,
     {
```

Comment:
When returning an array of elements, each one should have a unique `key`.

```suggestion
    {
      key: title,
```

https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key